package/src/lib/ObjectAssertion.ts

```diff
@@ -126,18 +126,50 @@ export class ObjectAssertion<T extends Struct> extends Assertion<T> {
       expected: keys,
       message: `Expected the object to contain at least one of the provided keys <${prettify(keys)}>`,
     });
-
     const invertedError = new AssertionError({
       actual: Object.keys(this.actual),
       message: `Expected the object NOT to contain any of the provided keys <${prettify(keys)}>`,
     });
+
     return this.execute({
       assertWhen: keys.some(key => this.hasOwnProp(key)),
       error,
       invertedError,
     });
   }
 
+  /**
+   * Check if the object has exactly the provided keys.
+   *
+   * @example
+   * ```
+   * expect({ x: 1, y: 2, z: 3 }).toHaveKeys("x", "y", "z");
+   * ```
+   *
+   * @param keys the keys the object should have
+   * @returns the assertion instance
+   */
+  public toHaveKeys(...keys: Array<keyof T>): this {
```

Comment:
Should we consider naming this something like "toExactlyHaveKeys" or similar to denote that it is an exact comparation?. I'm not sure about this though. Let me know what you think. :)

Reply:
Yeah, I thought about something similar, but it'd be a bit repetitive because we already have `.toContainKeys`, which are not exact. Also, I'd prefer keeping the names small and straightforward so they are easier to use and remember 🙂 

Let's keep it this way for now, and we can add an alias later if we think it's necessary.